codespell_lib/data/dictionary.txt

```diff
@@ -5111,6 +5111,7 @@ artice->article
 articel->article
 articels->articles
 artices->articles
+articifial->artificial
```

Comment:
```suggestion
articifial->artificial
articifially->artificially
```

Reply:
Every time I try to commit this with GitHub UI, I get some weird error I haven't seen before.  I have tried refreshing.

![screenshot of GitHub error](https://github.com/codespell-project/codespell/assets/8990777/831a7777-464e-49f0-b2d1-ce2fb48b5296)

I am just going to commit this locally

Reply:
Okay, it's commited